product_category_active/models/product_category.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ProductCategory(models.Model):
+    _inherit = "product.category"
+
+    active = fields.Boolean(
+        string="Active",
```

Comment:
Unfortunately there is another one. You always need to check that the pre-coomit ends up without error. In this case it's complaining about these 2 lines. The second one is redundant as the same string would be rendered just from the field's name.

So please remove the line 12, rerun pre-commit locally to be sure everything else is fine. Then commit, squash with the previous one and should be fine to go.